test/dict.jl

```diff
@@ -701,6 +701,79 @@ end
     @test_throws ArgumentError IdDict([1, 2, 3, 4])
     # test rethrow of error in ctor
     @test_throws DomainError   IdDict((sqrt(p[1]), sqrt(p[2])) for p in zip(-1:2, -1:2))
+
+    h = IdDict()
```

Comment:
These tests look like they will be pretty slow. Can you crank down the numbers?

Reply:
Yep sorry, I just blindly copied from the tests above.